backend/app/src/main/java/com/moh/phlat/backend/esb/boundary/PlrDataLoad.java

```diff
@@ -0,0 +1,64 @@
+package com.moh.phlat.backend.esb.boundary;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.moh.phlat.backend.esb.json.MaintainFacilityRequest;
+import com.moh.phlat.backend.esb.json.MaintainFacilityResponse;
+import com.moh.phlat.backend.esb.json.MaintainHdsRequest;
+import com.moh.phlat.backend.esb.json.MaintainHdsResponse;
+import com.moh.phlat.backend.esb.json.MaintainResults;
+import com.moh.phlat.backend.model.Control;
+import com.moh.phlat.backend.model.ProcessData;
+
+import lombok.Getter;
+
+@Component
+public class PlrDataLoad {
+	
+	private static final Logger logger = LoggerFactory.getLogger(PlrDataLoad.class);
```

Comment:
There's two cases I can think of that may want logging: one is in the case where facility load failed and so the HDS and OF loads could not be done for that record, and the other is when HDS fails and thus the OF could not be done. This is mainly to make it easier for Lindsay and team to evaluate if there's any missing facility/HDS records. I think we can at least keep it as is, and if down the line we find that it's not needed then I can remove it. Does that sound good, or do you think we should remove it now to clean it up and then reintroduce it if we find a use for it?

Reply:
I meant; it is unused variable. Either use it at place you think is suitable for logging or remove the variable as I could not see any usage in the code. - Unused variable.

Reply:
I'll take it out for now then, since I'm not 100% sure yet if I'll have use of it; if I need to log some things later in here, then I can simply add it back in